datalad/tests/test_config.py

```diff
@@ -102,7 +102,7 @@ def test_something(path, new_home):
     assert_equal(cfg.getint('something', 'myint'), 3)
     assert_equal(cfg.getbool('something', 'myint'), True)
     assert_equal(cfg.getbool('doesnot', 'exist', default=True), True)
-    assert_raises(TypeError, cfg.getbool, 'something', 'user')
+    assert_raises(ValueError, cfg.getbool, 'something', 'user')
```

Comment:
A change in the exception raised should not be going into maint.

Reply:
Words of wisdom ;) agreed -- I will adjust target to be `master`